interfaces/systemd/service.go

```diff
@@ -53,6 +57,18 @@ func (s *Service) String() string {
 	if s.ExecStop != "" {
 		fmt.Fprintf(&buf, "ExecStop=%s\n", s.ExecStop)
 	}
+	if s.Wants != "" {
+		fmt.Fprintf(&buf, "Wants=%s\n", s.Wants)
```

Comment:
Buf should have been a `strings.Builder` not `bytes.Buffer`

Reply:
Good point, I think this should go into a followup PR, This one is already huge